content/index.pl.md

```diff
@@ -0,0 +1,252 @@
+
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Treść strony" class="full" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+<span class="box-h box-h-simple box-h-full">Materiały pokrewne</span><br>
+{% include video-link.html title="Film wprowadzający do dostępności internetowej i standardów W3C <em>(4 minuty)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits/" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Spojrzenie na dostępność {#context}
+
+<blockquote class="pull">
+  <p>Siła Internetu tkwi w jego uniwersalności.<br />
+    Dostęp dla każdego bez względu na niepełnosprawność jest jego zasadniczym atrybutem.</p>
+  <footer><cite>Tim Berners-Lee, Dyrektor W3C i twórca World Wide Web</cite></footer>
+</blockquote>
+
+Internet został zasadniczo zaprojektowany tak, aby służył wszystkim, niezależnie od ich urządzeń, oprogramowania, języka, lokalizacji czy sprawności. Gdy Internet osiąga ten cel, jest dostępny dla osób z różnorodnym zakresem zdolności słuchowych, ruchowych, wzrokowych i poznawczych.
+
+Dlatego wpływ niepełnosprawności zmienia się radykalnie w Internecie, bowiem Internet usuwa bariery w komunikacji i uczestnictwie, z którymi wielu ludzi mierzy się w świecie fizycznym. Z drugiej strony, gdy strony internetowe, aplikacje, technologie lub narzędzia są źle zaprojektowane, mogą stawiać bariery uniemożliwiające ludziom korzystanie z Internetu.
+
+**Dostępność jest niezastąpiona dla programistów i organizacji, które chcą tworzyć wysokiej jakości strony i narzędzia internetowe, zamiast wykluczać ludzi z korzystania ze swoich produktów i usług.**
+
+## Czym jest dostępność internetowa {#what}
+
+Dostępność internetowa oznacza, że ​​strony internetowe, narzędzia i technologie są projektowane i rozwijane w taki sposób, by mogły z nich korzystać osoby z niepełnosprawnościami. Mówiąc dokładniej, ludzie mogą:
+
+-   postrzegać, rozumieć, nawigować i wchodzić w interakcję z Internetem
+-   współtworzyć Internet. 
+
+Dostępność internetowa obejmuje wszystkie rodzaje niepełnosprawności wpływające na dostęp do Internetu, w tym niepełnosprawność:
+
+-   słuchu,
+-   poznawczą,
+-   neurologiczną,
+-   fizyczną,
+-   mowy,
+-   wzroku.
+
+Dostępność Internetu przynosi korzyści także osobom _bez_ niepełnosprawności, na przykład:
+
+-   osobom korzystającym z telefonów komórkowych, inteligentnych zegarków, inteligentnych telewizorów i&nbsp;innych urządzeń z małymi ekranami, różnymi trybami wprowadzania danych itp.
+-   osobom starszym ze zmieniającymi się zdolnościami ze względu na starzenie się,
+-   osobom z „tymczasową niepełnosprawnością”, np. złamaną ręką lub zgubionymi okularami,
+-   osobom oganiczonym okolicznościami, np. jasnym światłem słonecznym lub w otoczeniu, w którym nie mogą słuchać dźwięku,
```

Comment:
Original content: "people with “situational limitations”"

@zwiastunsw Could the wording be closer to the original? This terminology has been chosen by the EOWG after [extensive discussion](https://www.w3.org/WAI/EO/wiki/Situational_terminology).

Reply:
```suggestion
-   osobom z ograniczeniami sytuacyjnymi, np. jasnym światłem słonecznym lub w otoczeniu, w którym nie mogą słuchać dźwięku,
```